packages/contracts/src/contracts/current/test/TestAssetProxyDispatcher/TestAssetProxyDispatcher.sol

```diff
@@ -0,0 +1,33 @@
+/*
+
+  Copyright 2018 ZeroEx Intl.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
+pragma solidity ^0.4.21;
+
+import "../../protocol/Exchange/MixinAssetProxyDispatcher.sol";
+
+contract TestAssetProxyDispatcher is MixinAssetProxyDispatcher {
+    function publicDispatchTransferFrom(
+        bytes memory assetMetadata,
+        address from,
+        address to,
+        uint256 amount)
+        public
+    {
+        dispatchTransferFrom(assetMetadata, from, to, amount);
+    }
```

Comment:
Why do we need this? Isn't `MixinAssetProxyDispatcher` going to be part of the Exchange contract?

Reply:
See comment above - we can't unit test this without wrapping it with a public function.